[BUG] Version 0.13.0 can't be installed because of a dependency on `@gillsdk/config-eslint`

<!-- Please provide all of the information requested below. Our maintainers are limited on time without all of this information it's not possible for us to help and your bug report will be closed. -->

In #252 a [direct dependency on @gillsdk/config-eslint got added](https://github.com/gillsdk/gill/pull/252/files#diff-1736f534c37aa7f1c90d2209dc4f90001dcc771dd345e088d96962bd23cefd9cR113).

This prevents `gill@0.13.0` from being installed 

**Which gill sdk packages are you having issues with?**

For example: gill

**What versions of these packages are you using?**

For example: v0.13.0

```

 ERR_PNPM_FETCH_404  GET https://registry.npmjs.org/@gillsdk%2Fconfig-eslint: Not Found - 404

This error happened while installing the dependencies of gill@0.13.0

@gillsdk/config-eslint is not in the npm registry, or you have no permission to fetch it.

```

--- a/.changeset/brown-crabs-film.md
+++ b/.changeset/brown-crabs-film.md
@@ -1,0 +1,5 @@
+---
+"gill": minor
+---
+
+move @gillsdk/config-eslint to devDependencies

--- a/packages/gill/package.json
+++ b/packages/gill/package.json
@@ -110,7 +110,6 @@
     "node": ">=20.18.0"
   },
   "dependencies": {
-    "@gillsdk/config-eslint": "workspace:*",
     "@solana-program/address-lookup-table": "^0.9.0",
     "@solana-program/compute-budget": "^0.10.0",
     "@solana-program/system": "^0.9.0",
@@ -119,6 +118,9 @@
     "@solana/codecs": "^5.0.0",
     "@solana/kit": "^5.0.0",
     "@solana/transaction-confirmation": "^5.0.0"
+  },
+  "devDependencies": {
+    "@gillsdk/config-eslint": "workspace:*"
   },
   "peerDependencies": {
     "typescript": ">=5"

